src/main/java/org/broadinstitute/hellbender/tools/spark/CreateHadoopBamSplittingIndex.java

```diff
@@ -17,37 +10,55 @@
 import org.broadinstitute.barclay.help.DocumentedFeature;
 import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
-import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.cmdline.programgroups.ReadDataProgramGroup;
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.io.IOUtils;
 import org.broadinstitute.hellbender.utils.read.ReadConstants;
 import org.codehaus.plexus.util.FileUtils;
 import org.seqdoop.hadoop_bam.SplittingBAMIndexer;
 
-import java.io.BufferedInputStream;
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-
-@CommandLineProgramProperties(summary = "Create a HadoopBam splitting index as well as a bam index from a bam file.",
-        oneLineSummary = "create a hadoop-bam splitting index" ,
-        programGroup = SparkProgramGroup.class)
+import java.io.*;
+
+/**
+ * Create a Hadoop BAM splitting index and optionally a BAM index from a BAM file.
+ *
```

Comment:
I'd like to know more about the significance of the 4096 splitting-index-granularity default value. For another time perhaps.